src/amber/router/context.cr

```diff
@@ -53,7 +53,7 @@ class HTTP::Server::Context
   end
 
   def request_handler
-    route.payload
+    route.payload.not_nil!
```

Comment:
A route should always match no?

Reply:
Yes because routes which don't match are aborted [over here](https://github.com/robacarp/amber/blob/ae786b5f79ee648dcec4b5fd6198b27bed9f8d9a/src/amber/pipes/pipeline.cr#L16).